serial_views.py

```diff
@@ -0,0 +1,153 @@
+# serialize data for all models
+from django.http import HttpResponse
+
+
+import WAM_APP_FRED.oep_models as oep_modles
+# from WAM_APP_FRED.db_sqla import *
+from shapely.wkb import loads as loadswkb
+from geojson import Feature, FeatureCollection, dumps
+from geoalchemy2 import functions
+import sqlahelper as sah
+from datetime import datetime
+
+# def serializer():
+#     """
+#     returns the queryed table id, geom as GEOJSON featureCollection
+#     serializer for table without relations
+#     :return: dict - geojson featureCollection
+#     """
+#
+#     #ToDO: after testing done change to input pram
+#     Session = sah.get_session()
+#     session = Session()
+#     ##############################################
+#
+#     features = []
+#
+#     for record in session.query(oep_modles.classes['Location']).limit(100):
+#         geometry = loads(str(record.point), True)
+#         propertys = record.id
+#         feature = Feature(id=record.id, geometry=geometry)
+#
+#         features.append(feature)
+#
+#     return dumps(FeatureCollection(features))
+
+class Serializer():
```

Comment:
Created the serializer as class so one can better understand what kind of data is in a GeoJSON that is returned by a class method.

Reply:
I am not sure what you mean here